webapp/components/admin_console/admin_navbar_dropdown.jsx

```diff
@@ -99,20 +103,21 @@ export default class AdminNavbarDropdown extends React.Component {
                         className='dropdown-menu'
                         role='menu'
                     >
+                        {teams}
                         <li>
                             <Link
                                 to={'/select_team'}
                             >
+                                <i className='fa fa-exchange'/>
                                 <FormattedMessage
                                     id='admin.nav.switch'
-                                    defaultMessage='Switch to {display_name}'
+                                    defaultMessage='Switch Teams'
                                     values={{
```

Comment:
If you removed the need for a value, then why not delete the `values` attr, it seems to me that is no longer needed.